src/ServiceControl.Audit.Persistence/IPersistenceConfiguration.cs

```diff
@@ -1,9 +1,12 @@
 ﻿namespace ServiceControl.Audit.Persistence
 {
+    using System.Threading;
+    using System.Threading.Tasks;
     using Microsoft.Extensions.DependencyInjection;
 
     public interface IPersistenceConfiguration
     {
-        void ConfigureServices(IServiceCollection serviceCollection, PersistenceSettings settings);
+        IPersistenceLifecycle ConfigureServices(IServiceCollection serviceCollection, PersistenceSettings settings);
```

Comment:
It feels like this needs another name now because it's doing more than just configure the services.